### ISSUE TITLE
No module named 'fairease.udal.udal'

### ISSUE THREAD

**Issue by ctroupin** [fair-ease/py-udal-fe-impl#6]
Hello,

I'm trying to run the notebook [example](https://github.com/fair-ease/py-udal-fe-impl/blob/main/examples/argo_data.ipynb) after cloning the project. 

The first cell execution gives the following message
```python
ModuleNotFoundError                       Traceback (most recent call last)
/tmp/ipykernel_80484/1992108308.py in <module>
      1 from pathlib import Path
----> 2 from fairease.udal.udal import UDAL, Config
      3 
      4 base_data_dir = './data/'
      5 config = Config()

ModuleNotFoundError: No module named 'fairease.udal.udal'
```

Is there a step I'm missing?
Thanks!

**Comment by jorgecunhamendes** [comment]
Hello @ctroupin. I think you did not install "py-udal-fe-impl" into the environment used to run the notebook. You can follow (part of) these instructions: https://github.com/fair-ease/py-udal-fe-impl?tab=readme-ov-file#development-setup

**Comment by jorgecunhamendes** [comment]
This should be solved alongside #7 .
